colin/cli/colin.py

```diff
@@ -151,5 +182,13 @@ def _print_checks(checks):
             click.echo(str(check))
 
 
+def _get_log_level(debug, verbose):
+    if debug:
+        return logging.DEBUG
+    elif verbose:
+        return logging.INFO
```

Comment:
Just wondering whether it makes sense to specify `--debug` and `--verbose` at the same time, I mean what about:

```python
if debug and verbose:
    raise click.BadOptionUsage("Options '--debug' and '--verbose' cannot be used together.")
```

Reply:
+1

Reply:
+1